### PR TITLE
Drop biz day for 2016

### PR DESCRIPTION
**Pull request by imsickofmaps**
It may return if we feel like it, or in 2017

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@vhata / @MarijeP - review please

File: schedule/index.html [hunk 1]
```diff
@@ -87,25 +87,15 @@
 <div class="row white-background">
   <div class="col-xs-12 col-md-12">
     <p>
-      ScaleConf runs over <strong>three days</strong>. We start with one day of
-      fireside chats focussing on <strong>the business of scale</strong>, while the
-      other two days are presentations from local and international experts on
+      ScaleConf 2016 runs over <strong>two days</strong> packeed with
```
Comment (adrianmoisey) [inline schedule/index.html]: 'packeed'

File: speak/index.html [hunk 2]
```diff
@@ -9,9 +9,7 @@
...
-
-        <p>There will be <strong>one day</strong> of business-oriented scaling talks, followed by <strong>two days</strong> of our usual technical talks. The Business of Scale day focuses on the issues leaders and their teams face when growing a technology-driven organisation. You'll get to hear from those who've faced the challenges of building engineering teams, growing healthy customer acquisition strategies, maintaining inspiring working environments, balancing customer demands and technical debt, dealing with exit opportunities and more.</p>
+        <p>ScaleConf will take place over two days in March 2016, in Cape Town, South Africa. It is a <strong>300-person</strong>, <strong>single-track</strong> conference, focusing on <strong>scalability and high availability</strong> on the internet today, the challenges faced when growing a company or service, and the technologies and methodologies we can use to mitigate these problems.</p>
```
Comment (adrianmoisey) [inline speak/index.html]: Nitpick: surely it's "Internet"?

Comment (adrianmoisey) [inline speak/index.html]: There are a few of these

**Pull request merged**